pkgs/development/tools/rust/cargo-embed/default.nix

```diff
@@ -1,22 +1,22 @@
 { lib
-, rustPlatform, fetchFromGitHub
-, libusb1, pkg-config, rustfmt }:
+, rustPlatform, fetchCrate
+, libusb1, libftdi1, pkg-config, rustfmt }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-embed";
-  version = "0.8.0";
+  version = "0.10.1";
 
-  src = fetchFromGitHub {
-    owner = "probe-rs";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "0klkgl7c42vhqxj6svw26lcr7rccq89bl17jn3p751x6281zvr35";
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "1s6qwsqrr9y7hdpn23k2vcic19j4zbwisbxyp8bi7qc68329hf44";
   };
```

Comment:
I would prefer fetching from GitHub.

Reply:
Not easily possible, the release is not tagged. I could pester upstream to push the tags, but it's easier to just fetch the crate, since that's the "official" source for the code anyways (it's normally installed through `cargo install`, which would fetch from crates.io and not github).

Reply:
> I could pester upstream to push the tags, but it's easier to just fetch the crate

Yes, please. Anything can be uploaded to crates.io pretending to be the release.

Reply:
> Anything can be uploaded to crates.io pretending to be the release.

Can you explain what you mean? Crates.io is the main way people install cargo extensions, and crates.io releases are immutable. So I'm not sure what the problem is. I'd like to understand the issue better.

Anyways, I made an issue upstream: https://github.com/probe-rs/cargo-embed/issues/219

Reply:
Actually crates.io should be used as source of truth instead of GH. Nobody other then me and @Tiwalun can release on crates.io. While there is more people on GH that can actually operate git.

In any way, I added the tags.